Help development with clean command and docs (#66)
athenianco/athenian-api: server/README.md

And open http://localhost:8080/v1/ui

If you want to run your own API image, use instead:
```
# Run the API container
IMAGE=your_api_image:tag make run-api
```

> You can clean API data, created from `make run-api`, by running:

Reviewer: ```suggestion
You can erase the API data fixtures created by `make run-api` with:
```
